products/knownprojects/bash/01_dataloading.sh

```diff
@@ -6,7 +6,7 @@ max_bg_procs 5
 if [ -n "${BUILD_ENGINE_SCHEMA}" ]; then
     echo "Dropping and creating build schema '$BUILD_ENGINE_SCHEMA'"
     run_sql_command "DROP SCHEMA IF EXISTS ${BUILD_ENGINE_SCHEMA} CASCADE;"
-    # run_sql_command "VACUUM (FULL);"
+    run_sql_command "VACUUM (ANALYZE);"
```

Comment:
build time was gradually increasing when building frequently and I think this is helping keep it low and consistent